inference-engine/ie_bridges/c/src/ie_c_api.cpp

```diff
@@ -335,6 +336,64 @@ IEStatusCode ie_core_read_network_from_memory(ie_core_t *core, const uint8_t *xm
     return status;
 }
 
+IEStatusCode ie_core_import_network(ie_core_t *core, const char *file_name, const char *device_name, \
+        const ie_config_t *config, ie_executable_network_t **exe_network) {
+    IEStatusCode status = IEStatusCode::OK;
+
+    if (core == nullptr || file_name == nullptr || device_name == nullptr || config == nullptr || exe_network == nullptr) {
```

Comment:
I think `config` can be `nullptr` if we don't want to configure network loading

Reply:
can we also fix other places?

Reply:
fixed